MetaMask/metamask-extension
# E2e revoke permissions

## **Description**



E2e test for new wallet_revokePermissions (current unmerged https://github.com/MetaMask/metamask-extension/pull/21948). Once that PR is merged the base branch will need to target `develop`, this PR will be taken out of draft, and ready for review.

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1706

## **Manual testing steps**

1. Pull down this branch
2. Run `yarn && yarn build:test && yarn test:e2e:chrome:rpc`
3. Should pass
4. Optional run `yarn test:e2e:single test/e2e/json-rpc/wallet_revokePermissions.spec.js --browser=chrome` should pass

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
